### PR TITLE
Move composer.json validate of sub-packages to GitHub Actions

### PR DESCRIPTION
**Pull request by glensc**
Example failure job output (2 packages):

```
[...]
Error: "packages/zend-acl/composer.json" does not contain valid JSON
Parse error on line 8:
...: ">=5.3.3"        "zf1s/zend-exception...
---------------------^
Expected one of: 'EOF', '}', ':', ',', ']'
                                                                 
  [Seld\JsonLint\ParsingException]                               
  "packages/zend-acl/composer.json" does not contain valid JSON  
  Parse error on line 8:                                         
  ...: ">=5.3.3"        "zf1s/zend-exception...                  
  ---------------------^                                         
  Expected one of: 'EOF', '}', ':', ',', ']'                     
                                                                 

packages/zend-amf/composer.json is valid
packages/zend-application/composer.json is valid
packages/zend-auth/composer.json is valid
[...]
packages/zend-service-twitter/composer.json is valid
packages/zend-service-windowsazure/composer.json is valid

Error: "packages/zend-service-yahoo/composer.json" does not contain valid JSON
Parse error on line 20:
...l adapters"        "zf1s/zend-validate"
---------------------^
Expected one of: 'EOF', '}', ':', ',', ']'
                                                                           
  [Seld\JsonLint\ParsingException]                                         
  "packages/zend-service-yahoo/composer.json" does not contain valid JSON  
  Parse error on line 20:                                                  
  ...l adapters"        "zf1s/zend-validate"                               
  ---------------------^                                                   
  Expected one of: 'EOF', '}', ':', ',', ']'                               
                                                                           

packages/zend-service/composer.json is valid
packages/zend-session/composer.json is valid
packages/zend-soap/composer.json is valid
[...]
packages/zend-xml/composer.json is valid
packages/zend-xmlrpc/composer.json is valid
Error: Process completed with exit code 1.
```
- Origin: https://github.com/zf1s/zf1/runs/1583880760?check_suite_focus=true

### CODE REVIEW

**Comment by glensc** [comment]
There isn't probably a point to do `composer validate` with each PHP version. but having it this way keeps the CI pipeline simple. So wdyt?

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -41,4 +42,10 @@ jobs:
           git ls-files '**/*.php' | while read f; do php -l "$f" > /dev/null || touch failed && echo -n .; done
           test ! -f failed
 
+      # https://github.com/zf1s/zf1/pull/6#issuecomment-495397170
+      - name: Validate composer.json is valid for all packages
```
Comment (glensc) [inline .github/workflows/tests.yml:46]: There isn't probably a point to do `composer validate` with each PHP version. but having it this way keeps the CI pipeline simple. So wdyt?

Comment (falkenhawk) [reply .github/workflows/tests.yml]: it's fine, thanks

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ jobs:
         with:
           php-version: ${{ matrix.php }}
           ini-values: display_errors=off, log_errors=on
+          extensions: :xdebug
```
Comment (falkenhawk) [inline .github/workflows/tests.yml:35]: adding `xdebug` is not relevant for the scope of this PR, is it?

Comment (glensc) [reply .github/workflows/tests.yml:35]: details are in the commit message.

**Review by falkenhawk — APPROVED** [review]

**Comment by falkenhawk** [comment]
Thank you @glensc for your contribution!

**Pull request merged**